pkg/cloud/ibmcloud/clients/machines.go

```diff
@@ -73,28 +76,34 @@ func NewInstanceServiceFromMachine(kubeClient kubernetes.Interface, machine *clu
 
 func (gs *GuestService) guestWaitReady(Id int) {
 	// Wait for transactions to finish
-	log.Printf("Waiting for transactions to complete before destroying.")
+	klog.Info("Waiting for transactions to complete before destroying.")
 	s := services.GetVirtualGuestService(gs.sess).Id(Id)
 
 	// Delay to allow transactions to be registered
 	time.Sleep(5 * time.Second)
 
+	// TODO: make it V(2) after initial development work done if necessary
+	//if klog.V(2) {
+	// Enable debug to show messages from IBM Cloud during node provision
+	s.Session.Debug = true
+	//}
 	for transactions, _ := s.GetActiveTransactions(); len(transactions) > 0; {
-		log.Print(".")
 		// TODO(gyliu513) make it configurable or use the notification mechanism to optimize
 		// the process instead of hardcoded waiting.
 		time.Sleep(5 * time.Second)
 		transactions, _ = s.GetActiveTransactions()
 	}
-	log.Println("wait done")
+	s.Session.Debug = false
+
+	klog.Info("Waiting for trasactions <%D> done.")
```

Comment:
can you help to fix this ?I didn't pay attention so I closed 2 commits same time

also a typo

Reply:
transaction